### PR TITLE
Minor typo fix in console alert message "it's" -> "its"

### PR DESCRIPTION
**Pull request by simonbromberg**

### CODE REVIEW

File: scripts/build-dependencies.sh [hunk 1]
```diff
@@ -23,4 +23,4 @@ then
     build_dependencies
 else
     alert_to_install_carthage
-fi
```
Comment (liuxuan30) [inline scripts/build-dependencies.sh]: will this matter? Sometimes I see such diff but not sure why introduced.

**Comment by danielgindi** [comment]
This is not a typo. Please learn English grammar...

**Comment by simonbromberg** [comment]
"it's" is a contraction of "it is"

"its" is a possessive determiner

So unless the sentence is supposed to read "In order to build Charts you need to use Carthage to build **it is** dependencies" then yes, **it's** a typo.

**Comment by danielgindi** [comment]
Well, my bad. I looked at the diff the other way around. Thought that the original was "its" :-)

**Pull request merged**